tests/test_backends.py

```diff
@@ -11,6 +11,13 @@
 
 class TestConsoleEmailBackend:
     def test_write_message(self):
+        msg = EmailMessage(body="foo\nbar")
+        with io.StringIO() as stream:
+            backends.ConsoleEmailBackend(stream=stream).write_message(msg)
+            stdout = stream.getvalue()
+            assert "foo\nbar" in stdout
+
+    def test_write_message__multi_alternatives(self):
```

Comment:
Maybe use single and multipart in the test names. That's more along the language used in this context.